tox.ini

```diff
@@ -168,13 +167,13 @@ commands = yamllint .
 [testenv:coveralls]
 envdir = {toxworkdir}/env-{env:TRAVIS_PYTHON_VERSION:coveralls}
 passenv = TRAVIS TRAVIS_*
-whitelist_externals = bash
 deps =
     coveralls
 changedir = {[base]changedir}
+whitelist_externals =
+    {[base]whitelist_externals}
 commands =
-    bash -c 'cd ..; bash .travis/fix-coverage.sh; cd -'
-    coveralls
+    bash {toxinidir}/.travis/runcoveralls.sh {posargs}
```

Comment:
Without changes to config.sh, this change will disable coveralls for the network role AFAIU. Even though it is currently broken, it should not be silently disabled.

Reply:
I thought `.travis/runcoveralls.sh` was supposed to do the same thing that
```
cd ..; bash .travis/fix-coverage.sh; cd -
coveralls
```
does.  If not, then @i386x what was the purpose of `.travis/runcoveralls.sh`?  @i386x did you have some other purpose/functionality intended for `.travis/runcoveralls.sh`?

Reply:
> Without changes to config.sh,

What changes need to be made to `network/.travis/config.sh` to enable coveralls, and can I include them in this PR?

Reply:
This change is there to allow disabling it (which is the default), so it needs something like `LSR_PUBLISH_COVERAGE=true` or `LSR_PUBLISH_COVERAGE=yes` not sure if there is a specific style for booleans in the template wrappers.